spec/integration/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -11,4 +11,99 @@ spec:
     wait_for_data_timeout:
       description: Timeout for waiting for metrics data during a system test.
       type: string
-      example: 10m
\ No newline at end of file
+      example: 10m
+    agent:
```

Comment:
I did not set any JSON patch since I thought it was better to allow all packages to define these keys. WDYT ?

Reply:
Yes, no need to add a json patch here.